tools/festive_moogle/give_items.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+import mysql.connector as mariadb
+
+# Change with correct info ##
+sqlusername = "darkstar"    #
+sqlpassword = "password"    #
+sqldatabase = "dspdb"       #
+#############################
+
+
```

Comment:
why would you even do this? its not hard to create char vars

Reply:
trying to be nice?
/shrug

Reply:
I can erase it if thats the ask =)

Reply:
someone else can decide if project wants this. ***not it***

Reply:
@whasf I thrust the python thingy onto you.

Reply:
silence is acceptance merging